wallet-new/src/Cardano/Wallet/WalletLayer.hs

```diff
@@ -1,3 +1,6 @@
+{-# LANGUAGE DeriveGeneric #-}
```

Comment:
Better to leave that file untouched for now. Ryan can most probably hook up all the errors in the old data structure without all the instances here. 
I apologize for the work done here, I've a bit overlooked stuff.